### PR TITLE
R4R: import rollup role to simplify devops

### PR DESCRIPTION
**Pull request by HaoyangLiu**
close https://github.com/mantlenetworkio/mantle/issues/605

### CODE REVIEW

File: l2geth/consensus/clique/scheduler.go [hunk 1]
```diff
@@ -87,17 +87,14 @@ type Scheduler struct {
...
 	log.Info("Create Sequencer Server")
 
 	syncer := synchronizer.NewSynchronizer()
 	schedulerAddr, err := syncer.GetSchedulerAddr()
 	if err != nil {
 		return nil, err
 	}
-	if schedulerAddr.String() != schedulerAddress.String() {
-		return nil, fmt.Errorf("scheduler address mismatch, schedulerAddr from L1 %s,schedulerAddr from config %s", schedulerAddr.String(), schedulerAddress.String())
-	}
```
Comment (Kukoomomo) [inline l2geth/consensus/clique/scheduler.go:100]: LGTM

File: l2geth/miner/worker.go [hunk 2]
```diff
@@ -20,6 +20,7 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+	"github.com/mantlenetworkio/mantle/l2geth/rollup"
```
Comment (Kukoomomo) [inline l2geth/miner/worker.go:23]: format

File: l2geth/cmd/utils/flags.go [hunk 3]
```diff
@@ -1176,11 +1171,20 @@ func setRollup(ctx *cli.Context, cfg *rollup.Config) {
...
+		str := ctx.GlobalString(RollupRoleFlag.Name)
+		switch str {
+		case "scheduler", "SCHEDULER":
+			cfg.RollupRole = rollup.SCHEDULER_NODE
+		case "sequencer", "SEQUENCER":
+			cfg.RollupRole = rollup.SEQUENCER_NODE
+		case "verifier", "VERIFIER":
+			cfg.RollupRole = rollup.VERIFIER_NODE
+		case "replica", "REPLICA":
+			cfg.RollupRole = rollup.SEQUENCER_NODE
```
Comment (HaoyangLiu) [inline l2geth/cmd/utils/flags.go:1184]: @Kukoomomo @curryxbo 
Does a `replica` node have any different behavior with `sequencer` node?

Comment (curryxbo) [reply l2geth/cmd/utils/flags.go]: The current behavior is similar to that of verifier, but the data source is different. sequencer can be used to replace replica later

**Review by curryxbo — APPROVED** [review]

**Review by Kukoomomo — APPROVED** [review]

**Pull request merged**